css/css-writing-modes/sizing-orthog-vrl-in-htb-013-ref.xht

```diff
@@ -18,6 +18,8 @@
     {
       height: 100%;
       writing-mode: vertical-rl;
+      overflow: hidden; /* since test and reference would place scrollbars on
```

Comment:
Maybe s/would place/might place/, throughout this commit?

(I don't think these styles do actually place scrollbars on opposite sides in any browsers right now; I think because the root scrollport has special behavior, in Firefox at least.  I suspect that's changing in WebKit in https://bugs.webkit.org/show_bug.cgi?id=224224 though.)

Reply:
Now am I uncertain whether I do want to change root scrollbar placement in that bug. :-)  If I do, I'll merge this PR.  If not, I'll abandon it.

Reply:
OK so I think I do want to change the placement, since we already change the placement for horizontal RTL documents.

Reply:
Yup, I noticed that. Makes sense. thanks!